transport/transport-backend-cct/src/test/java/com/google/android/datatransport/cct/CctTransportBackendTest.java

```diff
@@ -157,32 +137,55 @@ public void testCCTSuccessLoggingRequest() {
             .withHeader(
                 "User-Agent",
                 equalTo(String.format("datatransport/%s android/", BuildConfig.VERSION_NAME)))
-            .withHeader("Content-Type", equalTo("application/x-protobuf"))
-            .andMatching(batchRequestMatcher.test(batch -> batch.getLogRequestCount() == 1))
-            .andMatching(
-                firstLogRequestMatcher
-                    .test(r -> r.getRequestTimeMs() == wallClock.getTime())
-                    .test(r -> r.getRequestUptimeMs() == uptimeClock.getTime())
-                    .test(r -> r.getLogEventCount() == 1))
```

Comment:
@vkryachko how did this test pass? There are two events in the request, and we test them below.

Reply:
It definitely should not have passed, will take a closer look, but your test has the right assert( == 2)